packages/core/server/admin-api/services/query-layer-decorator.ts

```diff
@@ -186,11 +193,13 @@ const decorator = (service: IDecoratedService) => ({
       localizations: undefined,
     };
 
-    // If a URL alias is allready present, fetch it.
-    if (opts.data.url_alias) {
+    // @ts-ignore
+    if (opts.data.url_alias?.length) {
       urlAliasEntity = await getPluginService('urlAliasService').findOne(opts.data.url_alias);
```

Comment:
Just wondering if this statement still works as `opts.data.url_alias` is now an array.